contents/blog/how-we-built-email-onboarding.md

```diff
@@ -0,0 +1,132 @@
+---
+date: 2024-02-05
+title: "How we built our onboarding email flow (with actual performance data)"
+rootPage: /blog
+category: Inside PostHog
+author: ["joe-martin"]
+featuredImage: ../images/blog/blog_onboarding.jpg
+featuredImageType: full
+---
+
+Email is a powerful tool for getting users to engage with your product, but it can be  difficult to know how the best companies structure their onboarding email flows. You end up having to sign up to a bunch of them, then waiting and letting your inbox get flooded. 
```

Comment:
```suggestion
Email is a powerful tool for getting users to engage with your product, but it can be difficult to know how the best companies structure their onboarding email flows. You end up having to sign up to a bunch of them, then waiting and letting your inbox get flooded. 
```